File: src/.scripts/plotlens.m
clear;
a = load('a.txt');

y = a(:,1);
x = a(:,3:end);

positives = x(y == 1,:);
negatives = x(y == -1,:);

plot(negatives', '^b');
plot(positives(1:100,:)', 'pr')
axis([-1 10 -1 10]);

File: .scripts/plotlens.py
import numpy as np
import matplotlib.pyplot as plt

a = np.loadtxt("a.txt")
y = a[:,0]
x = a[:,2:]

positives = x[y == 1,:]
negatives = x[y == -1,:]

howmany = 100
plt.plot(negatives[:howmany,:].transpose(), "ro")
plt.plot(positives[:howmany,:].transpose(), "b+")
plt.axis([-2, 10, -2, 10])
plt.show()

